ts/navigation/AppStackNavigator.tsx

```diff
@@ -93,6 +96,12 @@ const InnerNavigationContainer = (props: InnerNavigationContainerProps) => {
     isNewPaymentSectionEnabledSelector
   );
 
+  // ITW Flags
+  const isItwTrialActive = useIOSelector(isItwTrialActiveSelector);
+  const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
+  const isItwEnabled = useIOSelector(isItwEnabledSelector);
+  const canBeActivated = isItwTrialActive && !isItwValid && isItwEnabled;
```

Comment:
isItwValid shouldn't be true?

Reply:
> isItwValid shouldn't be true?

`isItwValid` is `true` when the user already has an `eID` thus there's no need to activate the wallet again. I'm flipping it so this only works when the wallet doesn't have an `eID`